project/Doc.scala

```diff
@@ -47,7 +49,7 @@ object Scaladoc extends AutoPlugin {
       "-groups",
       "-doc-source-url", urlString,
       "-sourcepath", base.getAbsolutePath,
-      "-skip-packages", "akka.pattern"
+      "-skip-packages", "akka.pattern:org.specs2"
```

Comment:
Ouch! Could you please add a comment that includes https://issues.scala-lang.org/browse/SI-10028? Thanks!